fat/fat2/transactionbatch.go

```diff
@@ -69,24 +85,38 @@ func (t TransactionBatch) String() string {
 	return string(data)
 }
 
-// UnmarshalEntry unmarshals the Entry content as a TransactionBatch
-func (t *TransactionBatch) UnmarshalEntry() error {
-	return t.Entry.UnmarshalEntry(t)
-}
-
-// MarshalEntry marshals the TransactionBatch into the entry content
-func (t *TransactionBatch) MarshalEntry() error {
-	return t.Entry.MarshalEntry(t)
+//// UnmarshalEntry unmarshals the Entry content as a TransactionBatch
+//func (t *TransactionBatch) UnmarshalEntry() error {
+//	return t.Entry.UnmarshalEntry(t)
+//}
+//
+//// MarshalEntry marshals the TransactionBatch into the entry content
+//func (t *TransactionBatch) MarshalEntry() error {
+//	return t.Entry.MarshalEntry(t)
+//}
```

Comment:
😹

Reply:
I'll delete it. Adam removed these from the other fat token types.